media/css/m24/feature.scss

```diff
@@ -0,0 +1,93 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+
+// feature
+.m24-c-feature {
+    @include container;
+    padding-bottom: $spacer-xl;
+}
+
+.m24-c-feature-media {
+    padding-top: $container-padding;
+    padding-bottom: $spacer-xl;
+}
+
+.m24-c-feature-title {
+    margin-bottom: $spacer-xl;
+    font-size: $alias-text-title-h2;
+    line-height: 1;
+    text-wrap: balance;
+    text-wrap-style: balance;
+}
+
+.m24-c-feature-cta {
+    text-align: end;
+}
+
+@media #{$mq-xl} {
+    .m24-c-feature-title {
+        max-width: grid(9);
+    }
+}
+
+// columns
+
+.m24-t-columns-four {
+    align-items: flex-start;
+    gap: $spacer-md $grid-gutter;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    margin-bottom: 0;
+
+    @media #{$mq-lg} {
+        flex-flow: row wrap;
+    }
+
+    @media #{$mq-xl} {
+        flex-flow: row nowrap;
+    }
+}
+
+// picto
+
+.mzp-c-picto {
```

Comment:
The picto code is mostly the same as before I separated the files. I switched to using the `grid()` function to calculate max-width.